targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/common_rtc.h

```diff
@@ -0,0 +1,53 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015 ARM Limited
```

Comment:
add SPDX identifier and fix the year (new files 2018), also the company name

```
 * Copyright (c) 2018, Arm Limited and affiliates.
 * SPDX-License-Identifier: Apache-2.0
```

Reply:
Is the following commit appropriate:  bc4aed00cb6c6ef66f566518e4cba164e0c5e298 ?  I've excluded any old files we didn't touch such as itm_api.c.  Would you want the company name and SPDX added for those files (and not touch the date)?   I'll then follow this model for any other Arm source files located under the TARGET_NORDIC tree.

Reply:
Please follow for any, would be good 👍 

Thanks